.eslintrc.js

```diff
@@ -70,7 +70,7 @@ module.exports = {
         "@typescript-eslint/no-unsafe-call": "off",
         "@typescript-eslint/no-unsafe-member-access": "off",
         "@typescript-eslint/no-unsafe-return": "off",
-        "@typescript-eslint/no-unused-vars": "error",
+        "@typescript-eslint/no-unused-vars": ["error", { "ignoreRestSiblings": true }],
```

Comment:
Allows clever removal of properties from an object with destructuring. I'm open to debate on it, although to me it feels restricted when the intent of this type of code is quite clear. See [example below](https://github.com/Exygy/askdarcel-web/pull/65/files#diff-5ec4326e0f2461613ffd060db4b2a584c472a74f231e85d2df6794412beb1814R21).

Reply:
This approach works for me!